client/src/components/ChefSearch.js

```diff
@@ -25,13 +25,16 @@ const ChefSearch = ({ coords }) => {
   const [distanceFilter, setDistanceFilter] = useState(null);
 
   const classes = useStyles();
+
   useEffect(() => {
-    searchChefs();
-    // to change later
     (async function getCuisines() {
       const { data: allCuisines } = await API.get("/cuisines");
       setCuisineTypes(allCuisines.map((c) => c.name));
     })();
+  }, []);
+
+  useEffect(() => {
+    searchChefs();
   }, [userAddress, userCoordinates, selectedCuisines, distanceFilter]);
```

Comment:
the dependency array here will throw warnings with eslint. its got stuff in the dependency array that are not in the useEffect hook.  @giftofgrub do you have eslint extension installed for vscode?